frontend/src/assets/utils/ImageWrapper.tsx

```diff
@@ -0,0 +1,7 @@
+import { ImgHTMLAttributes } from 'react';
+
+function ImageWrapper(src: string) {
+  return (props: ImgHTMLAttributes<HTMLImageElement>) => <img {...props} />;
+}
```

Comment:
👍

그런데 props 로 받는 src 를 넣어주지 않고 있는데 어떻게 전달되는건가용??

Reply:
근데 저희 컨벤션이 arrow function 이라 이렇게 사용해보는건 어덜까요?
```suggestion
const ImageWrapper = (src: string) => 
  (props: ImgHTMLAttributes<HTMLImageElement>) => <img src={src} {...props} />;
```

Reply:
>  어떻게 전달되는건가용??
오 그러네요 세심한 지적 감사합니다 !

> arrow function
적용했습니다 !